vault/dendron.topic.publish.cook.common.md

```diff
@@ -19,29 +19,29 @@ npm install --save @dendronhq/dendron-cli
     ```
 1. Clone the nextjs repository and install dependencies
     ```sh
-    dendron publish init
+    npx dendron publish init
     ```
 
 ### Build Notes
 - NOTE: run this command inside your [[workspace|dendron.topic.workspace]]
 ```sh
-dendron publish build
+npx dendron publish build
```

Comment:
This makes the [Publish CLI](https://wiki.dendron.so/notes/yQVhJtdQ40n3SLHJKAeeU/) page confusing as if an npx one-liner would work without context in the referenced page.

`npx` would not know that `dendron` is from @dendronhq/dendron-cli if it's not installed.

Reply:
Agreed. This is what I meant when I commented above.

> Wouldn't it be great to document both options - if it is globally installed or locally inside working directory. It would cater to both type of audience.